### PR TITLE
v2v: add TC 'Improve error "No module named ovirtsdk4" in v2v rhv-upl…

### PR DESCRIPTION
**Pull request by vwu-vera**
Please check bz#1601943

### CODE REVIEW

**Comment by vwu-vera** [comment]
@xiaodwan Please help to review. Thanks a lot.:)

File: v2v/tests/src/function_test_esx.py [hunk 1]
```diff
@@ -799,6 +799,8 @@ def vm_check(status_error):
             LOG.info('Set http_proxy=%s, https_proxy=%s', http_proxy, https_proxy)
             os.environ['http_proxy'] = http_proxy
             os.environ['https_proxy'] = https_proxy
+        if 'ovirtsdk4_pkg' in checkpoint:
+            utils_package.package_remove(['python3-ovirt-engine-sdk4'])
```
Comment (xiaodwan) [inline v2v/tests/src/function_test_esx.py:803]: This way is dangerous, if the repo is configured properly, the package will not be installed back, then following cases will
be blocked.

Comment (xiaodwan) [reply v2v/tests/src/function_test_esx.py:803]: I prefer to find the ovirtsdk4 directory's path first, then rename the directory.

Example:
1) find the path
>>> import ovirtsdk4; import os.path; os.path.dirname(ovirtsdk4.__file__)
'/usr/lib64/python3.6/site-packages/ovirtsdk4'
2) rename the path
os.rename(src, dst)
3) do v2v convertion
4) recover the directory's name.
os.rename(dst, src)

Comment (xiaodwan) [reply v2v/tests/src/function_test_esx.py:803]: Compared to uninstall/reinstall a package, this way is more stable.  os.rename rarely fails.

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.negative_test.no_ovirtsdk4_pkg.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (113.60 s)

**Comment by vwu-vera** [comment]
@xiaodwan Thanks for the suggestions. For the auto testing, it is better to avoid changing the testing env. But in this case, this way is indeed better than prior one. Please help to review. :)

File: v2v/tests/src/function_test_esx.py [hunk 2]
```diff
@@ -799,7 +800,10 @@ def vm_check(status_error):
             LOG.info('Set http_proxy=%s, https_proxy=%s', http_proxy, https_proxy)
             os.environ['http_proxy'] = http_proxy
             os.environ['https_proxy'] = https_proxy
-
+        if 'ovirtsdk4_pkg' in checkpoint:
+            ovirt4_path = os.path.dirname(ovirtsdk4.__file__)
+            dst_ovirt4_path = ovirt4_path+'.bak'
```
Comment (xiaodwan) [inline v2v/tests/src/function_test_esx.py:805]: Please update the format of the expression.   
         dst_ovirt4_path = ovirt4_path + '.bak'

The others look fine.

Comment (vwu-vera) [reply v2v/tests/src/function_test_esx.py]: Correct the format. Thanks.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**